### DISCUSSION TITLE
Update Nuxt Image FAQ

### DISCUSSION THREAD

**Discussion by P4sca1** [Baroshem/nuxt-security#496] [General]
I think the Nuxt Image FAQ (https://nuxt-security.vercel.app/documentation/advanced/faq#using-nonce-with-csp-for-nuxt-image) should be updated. AFAIK, adding a nonce to an image tag doesn't do anything, because nonces are not supported for inline event handlers.
So right now using `NuxtImg` with strict CSP will always error during SSR, when there is an error loading the image.
See https://github.com/nuxt/image/issues/1011#issuecomment-2242761992 for more context.

**Comment by Baroshem** [comment]
Hey @P4sca1 

Thanks for this info! Would you be interested in updating the documentation to match this new state? I can provide all the help needed :)
